src/hotkey/hotkey_item.cpp

```diff
@@ -389,11 +396,13 @@ void clear_hotkeys(const std::string& command)
 
 void clear_hotkeys()
 {
+	lock_guard<mutex> lock(hotkeys_mutex_);
 	hotkeys_.clear();
 }
 
 const hotkey_ptr get_hotkey(const SDL_Event &event)
 {
+	lock_guard<mutex> lock(hotkeys_mutex_);
```

Comment:
This function returns a pointer, which is accessible after the lock is released.

Reply:
The hotkey pointers should still be okay, as far as i understand it it's just access to the hotkey list itself that is problematic.

Reply:
You're correct, sorry.